pkg/sql/create_view.go

```diff
@@ -408,7 +408,7 @@ func makeViewTableDesc(
 	}
 
 	if sc != nil {
-		sequenceReplacedQuery, err := replaceSeqNamesWithIDs(ctx, sc, viewQuery)
+		sequenceReplacedQuery, err := replaceSeqNamesWithIDs(ctx, sc, viewQuery, false)
```

Comment:
Annotate the Boolean with a comment indicating the parameter name as per the style guide